apps/openmw/mwworld/datetimemanager.cpp

```diff
@@ -0,0 +1,218 @@
+#include "datetimemanager.hpp"
+
+#include "../mwbase/environment.hpp"
+#include "../mwbase/world.hpp"
+
+#include "esmstore.hpp"
+#include "globals.hpp"
+#include "timestamp.hpp"
+
+namespace MWWorld
+{
+    int getDaysPerMonth(int month)
```

Comment:
Put declaration into header or move into anonymous namespace/make static to avoid possible ODR violation.